spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/BinaryTreeLSTM.scala

```diff
@@ -405,14 +407,73 @@ class BinaryTreeLSTM[T: ClassTag](
   }
 }
 
-object BinaryTreeLSTM {
+object BinaryTreeLSTM extends ModuleSerializable {
   def apply[@specialized(Float, Double) T: ClassTag](
     inputSize: Int,
     hiddenSize: Int,
     gateOutput: Boolean = true,
     withGraph: Boolean = true
   )(implicit ev: TensorNumeric[T]): BinaryTreeLSTM[T] =
     new BinaryTreeLSTM[T](inputSize, hiddenSize, gateOutput, withGraph)
+
+  override def loadModule[T: ClassTag](model : BigDLModule)
+                                      (implicit ev: TensorNumeric[T]) : ModuleData[T] = {
+    val moduleData = super.loadModule(model)
+    val binaryTreeLSTMModule = moduleData.module.asInstanceOf[BinaryTreeLSTM[T]]
+    binaryTreeLSTMModule.composers.clear
```

Comment:
it's to remove the auto-created ones after initialization and reassign to serialized ones